test/Microsoft.ML.TestFramework/Microsoft.ML.TestFramework.csproj

```diff
@@ -18,4 +18,8 @@
     <NativeAssemblyReference Include="LdaNative" />
     <NativeAssemblyReference Include="FastTreeNative" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Memory" Version="4.5.1" />
```

Comment:
This shouldn't be necessary.  All our test projects are netcoreapp2.1, and so Span and ReadOnlyMemory should be available by default.